frontend/src/models/api/Property.ts

```diff
@@ -0,0 +1,42 @@
+import { Api_Address } from './Address';
+import { Api_ConcurrentVersion } from './ConcurrentVersion';
+import Api_TypeCode from './TypeCode';
+
+export interface Api_Property extends Api_ConcurrentVersion {
+  id: number;
+  pid?: string;
+  pin?: number;
+  zoning?: string;
+  municipalZoning?: string;
+  notes?: string;
+
+  name?: string;
+  description?: string;
+  isSensitive: boolean;
+  isProvincialPublicHwy?: boolean;
+
+  latitude?: number;
+  longitude?: number;
+
+  landArea?: number;
+  areaUnit?: Api_TypeCode<string>;
+
+  isVolumetricParcel?: boolean;
+  volumetricMeasurement?: number;
+  volumetricUnit?: Api_TypeCode<string>;
+  volumetricType?: Api_TypeCode<string>;
+
+  propertyType: Api_TypeCode<string>;
+  status?: Api_TypeCode<string>;
+
+  // multi-selects
+  anomalies?: Api_TypeCode<string>[];
+  tenure: Api_TypeCode<string>[];
+  roadType?: Api_TypeCode<string>[];
+  adjacentLand?: Api_TypeCode<string>[];
+
+  dataSource?: Api_TypeCode<string>;
+  dataSourceEffectiveDate?: string;
+
+  address: Api_Address;
```

Comment:
address  could be undefined

Reply:
![image](https://user-images.githubusercontent.com/24322224/160694047-ff505bcf-10e4-4319-8db4-3b647a8aef48.png)

ADDRESS foreign key is required (not null) on the database so there won't be an instance of a property without address

Reply:
Unless DB schema changes, then yes we can modify the frontend to match (if/when it happens). I think it makes sense as all properties imported into the system need a valid address

Reply:
From the DB point of view yes. But from the API point of view everything is nullable